resourcepolicies.md

```diff
@@ -10,34 +10,298 @@ As we don't have yet an use case to iterate over all the existent resource polic
 
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators, users with ADMIN right on the target resource, users mentioned in the policy (eperson or member of the group) can access the resourcepolicy
+* 404 Not found - if the resourcepolicy doesn't exist (or was already deleted)
+
+### Search methods
+#### resource
+**/api/authz/resourcepolicies/search/resource?uuid=<:uuid>[&action=<:action>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the resource object of the policy
+* action: optional, limit the returned policies to the specified action (see GET description for allowed values)
+
+It returns the list of matching resource policies
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators and users with ADMIN right on the target resource can use the endpoint
+* 422 UNPROCESSABLE ENTITY - if the uuid parameter is missing or invalid
+
+#### eperson
+**/api/authz/resourcepolicies/search/eperson?uuid=<:uuid>[&resource=<:resource>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the eperson that benefit of the policy
+* resource: optional, limit the returned policies to the ones related to the specified resource
+
+It returns the list of matching resource policies
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators, the user specified in the uuid parameter or user with ADMIN permission over such user can use the endpoint
+* 422 UNPROCESSABLE ENTITY - if the uuid parameter is missing or invalid
+
+#### group
+**/api/authz/resourcepolicies/search/group?uuid=<:uuid>[&resource=<:resource>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the eperson that benefit of the policy
+* resource: optional, limit the returned policies to the ones related to the specified resource
+
+It returns the list of matching resource policies
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators, users member of the group specified in the uuid parameter or user with ADMIN permission over it can use the endpoint
+* 422 UNPROCESSABLE ENTITY - if the uuid parameter is missing or invalid
+
+#### authorizeAnyOf
+**/api/authz/resourcepolicies/search/authorizeAnyOf?resource=<:resource>&action=<:action>&action=<:actionN>[&useInheritance=<:boolean>]**
```

Comment:
For consistency (with other areas of this doc), the `resource` param should note it's a UUID.  For example `resource=<:uuid>`